PubMed.js

```diff
@@ -1284,59 +979,16 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"notes": [],
-				"tags": [
-					"Pancreatitis"
-				],
-				"seeAlso": [],
-				"attachments": [
-					{
-						"title": "PubMed entry",
-						"mimeType": "text/html",
-						"snapshot": false
-					}
-				],
+				"date": "Oct 1950",
 				"ISSN": "0025-7044",
-				"journalAbbreviation": "J Med Assoc State Ala",
+				"extra": "PMID: 14779137",
 				"issue": "4",
+				"journalAbbreviation": "J Med Assoc State Ala",
 				"language": "eng",
-				"extra": "PMID: 14779137",
-				"libraryCatalog": "NCBI PubMed",
-				"title": "Surgical management of pancreatitis",
+				"libraryCatalog": "PubMed",
 				"pages": "118-128",
 				"publicationTitle": "Journal of the Medical Association of the State of Alabama",
 				"volume": "20",
-				"date": "Oct 1950"
-			}
-		]
-	},
-	{
-		"type": "search",
-		"input": {
-			"PMID": "20729678"
-		},
-		"items": [
```

Comment:
No, not currently possible in Scaffold, but that's something that we would want to fix.

In the mean time, you can run tests in Firefox by going to chrome://zotero/content/tools/testTranslators/testTranslators.html